research/src/pages/focusgroup/focusgroup.explainer.mdx

```diff
@@ -361,48 +361,50 @@ When extending a focusgroup, traversal order is based on document order. Given t
 Example 6:
 
 ```html
-<div focusgroup>
-  <div id="A" tabindex="0"></div>
-  <div id="B" tabindex="-1">
-    <div>
-      <div focusgroup="extend">
-        <div id="B1" tabindex="-1"></div>
-        <div id="B2" tabindex="-1"></div>
+<div role=toolbar focusgroup>
+  <button id="A" type=button tabindex="0"></button>
+  <button id="B" type=button tabindex="-1"></button>
+  <div>
+      <div role=radiogroup focusgroup="extend">
+        <button id="B1" role=radio type=button tabindex="-1"></button>
+        <button id="B2" role=radio type=button tabindex="-1"></button>
```

Comment:
Part of me wants to mention that we probably _shouldn't_ add a role here to button and instead keep it a div. If the intent is OOTB illustration using just HTML, I don't know that radiogroup would yet fit into that...just a note :)

Reply:
I'm OK going back to a div. (I think I borrowed this particular arrangement from an ARIA authoring practices demo, but you have a point.)

Reply:
Fixed in https://github.com/openui/open-ui/pull/550/commits/cc22a4a335754d510e3e42a4acf885821db0f45d